tests/unittests/test_net.py

```diff
@@ -5373,21 +5373,20 @@ def test_select_none_found_raises(self, m_eni_avail, m_sysc_avail):
                           priority=['sysconfig', 'eni'])
 
     @mock.patch("cloudinit.net.sysconfig.available_sysconfig")
-    @mock.patch("cloudinit.util.get_linux_distro")
-    def test_sysconfig_available_uses_variant_mapping(self, m_distro, m_avail):
+    @mock.patch("cloudinit.util.system_info")
+    def test_sysconfig_available_uses_variant_mapping(self, m_info, m_avail):
```

Comment:
This test was checking some of the "variant" determining behavior, so I moved the bit of code into it's own function and am testing that separately now.

Reply:
Thx for the explanation here.